README.md

```diff
@@ -6,7 +6,136 @@ This package contains methods providing Saleor business logic for a storefront a
 
 > :warning: **Note: Saleor SDK is still under heavy development, and its API may change.**
 
-## Configuration
+## Table of Contents
+
+- [Setup](#setup)
+- [Features](#features)
+- [Local development](#local-development)
+
+## Setup
+
+There are two ways to use SDK - making custom implementation or using React components and hooks, which already has that implementation ready.
+
+### Using React
+
+First install SDK as dependency to your project
+
+```bash
+npm install @saleor/sdk
+```
+
+Use `SaleorProvider` with passed Saleor's client created by `createSaleorClient` in a prop. Then use React hooks in any component passed as child to `SaleorProvider`.
+
+```tsx
+import { SaleorProvider, createSaleorClient, useAuth, useAuthState } from "@saleor/sdk";
+
+const client = createSaleorClient({
+  apiUrl: "http://localhost:8000/graphql/,
+  channel: "",
+});
+
+const rootElement = document.getElementById("root");
+ReactDOM.render(
+  <SaleorProvider client={client}>
+    <App />
+  </SaleorProvider>,
+  rootElement
+);
+
+const App = () => {
+  const { login } = useAuth();
+  const { authenticated, user } = useAuthState();
+
+  const handleSignIn = async () => {
+    const { data } = await login({email: "admin@example.com", password: "admin"});
+
+    if (data.tokenCreate.errors.length) {
+      /**
+       * Unable to sign in.
+       **/
+    } else if (data) {
+      /**
+       * User signed in successfully.
+       **/
+    }
+  };
+
+  if (authenticated && user) {
+    return <span>Signed in as {user.firstName}</span>;
+  } else {
+    return <button onClick={handleSignIn}>Sign in</button>;
+  }
+};
+```
+
+### Custom implementation
```

Comment:
```suggestion
### Using with NodeJS and other frameworks
```

WDYT? "Custom implementation" sounds to me more like SDK modification, not using it

Reply:
sure, changed